templates/blog/article.html

```diff
@@ -18,7 +22,7 @@ <h1>{{ article.title.rendered|safe }}</h1>
       <div class="col-8">
         <div class="p-media-object">
           {% if article.author %}
-            <img src="{% if article.author.id == 217 %}{{ ASSET_SERVER_URL }}60d9b81e-picto-canonical.svg{% else %}{{ article.author.avatar_urls.96 }}{% endif %}" class="p-media-object__image is-round">
+            <img src="{% if article.author.id == 217 %}https://assets.ubuntu.com/v1/60d9b81e-picto-canonical.svg{% else %}{{ article.author.avatar_urls.96 }}{% endif %}" class="p-media-object__image is-round">
```

Comment:
`article.author.avatar_urls.96` doesn't seem to work, though `article.author.avatar_urls['96']` does.